crates/astria-sequencer-relayer/src/relayer/write/mod.rs

```diff
@@ -96,8 +96,8 @@ impl BlobSubmitterHandle {
     pub(super) fn try_send(
         &self,
         block: SequencerBlock,
```

Comment:
I mentioned it for `conductor` as well: IMO you could just make the channel be over `Box<SequencerBlock>`, which is a reasonable thing to do (so that the channel doesn't hold very large structures itself).

You would then not need to box every error variant individually.

Reply:
Done in d113119 :)